hera_pspec/pspecdata.py

```diff
@@ -291,28 +291,57 @@ def set_iC(self, d):
             self.iC().
         """
         for k in d: self._iC[k] = d[k]
-    
-    def q_hat(self, key1, key2, use_identity=False, use_fft=True, 
-              taper='none'):
+
+    def set_R(self, R_matrix):
+        """
+        Set the weighting matrix R for later use in q_hat.
+
+        Parameters
+        ----------
+        R_matrix : string or matrix
+            If set to "identity", sets R = I
+            If set to "iC", sets R = C^-1
+            Otherwise, accepts a user inputted dictionary
+        """
+
+        if R_matrix == "identity":
+            self._R = self.I
+        elif R_matrix == "iC":
+            self._R = self._iC
+        else:
+            self._R = R_matrix
+
+    def set_Qs(self, taper='none'):
```

Comment:
Yeah, it's tricky. I suppose we could put in an "if" statement so that if Q does not exist, it's computed. But then to do that, you would need to know the taper, which defeats the purpose of the reorganization.

Ok, I'm convinced. If the user is using PSpecData.pspec() anyway, they shouldn't have the opportunity to specify the taper in multiple locations anyway. I'll go back to the old way of doing things.